--- httrack-3.32.03/html/server/div/WebHTTrack-Websites.desktop~	2004-09-20 10:50:45.000000000 +0200
+++ httrack-3.32.03/html/server/div/WebHTTrack-Websites.desktop	2004-09-20 10:50:45.000000000 +0200
@@ -1,10 +1,12 @@
 [Desktop Entry]
+Encoding=UTF-8
 Version=1.0
 Type=Application
 Categories=Application;Network
 Terminal=false
-Name=Browse Mirrored Websites
+Name=WebHTTrack Browser
 Comment=Browse Websites Mirrored by WebHTTrack
+Comment[pl]=Przeglądanie stron zmirrorowanych przez WebHTTracka
 Exec=webhttrack browse
 Icon=/usr/share/httrack/icons/webhttrack.xpm
 Terminal=false
--- httrack-3.32.03/html/server/div/WebHTTrack.desktop~	2004-09-20 10:50:46.000000000 +0200
+++ httrack-3.32.03/html/server/div/WebHTTrack.desktop	2004-09-20 10:50:46.000000000 +0200
@@ -1,10 +1,12 @@
 [Desktop Entry]
+Encoding=UTF-8
 Version=1.0
 Type=Application
 Categories=Application;Network
 Terminal=false
 Name=WebHTTrack Website Copier
 Comment=Copy websites to your computer
+Comment[pl]=Kopiowanie stron na lokalny komputer
 Exec=webhttrack
 Icon=/usr/share/httrack/icons/webhttrack.xpm
 Terminal=false
